lib/json_schema_form/umm_form.rb

```diff
@@ -298,6 +298,36 @@ def accordion_id
       top_key.underscore.dasherize
     end
   end
+
```

Comment:
Style feedback for the below changes is appreciated. As I write this I think it'd be helpful to add comments to the explain each argument in the conditionals. Let me know if you think this would help.